### DISCUSSION TITLE
只能获取到USB设备吗？我的串口是485控制接口，好像不能够获取到

### DISCUSSION THREAD

**Discussion by hxb06** [LUJIAN2020/UsbSerialForAndroid.Net#8] [Q&A] [answered]
大佬您好，我是新手，做Android app开发发现尝试了Java很痛库，引入一些包之类的实在不是很懂，于是还是选择了用maui做app开发，demo我测试了可以通过USB做串口通信，但是我的安卓一体机是RS485控制口，它获取不到这个设备列表怎么办？

**Comment by LUJIAN2020** [comment, marked as answer]
> 大佬您好，我是新手，做Android app开发发现尝试了Java很痛库，引入一些包之类的实在不是很懂，于是还是选择了用maui做app开发，demo我测试了可以通过USB做串口通信，但是我的安卓一体机是RS485控制口，它获取不到这个设备列表怎么办？

可能需要用到android-serialport-api https://github.com/licheedev/Android-SerialPort-API
